account_invoice_ubl_mass_download/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Coop IT Easy SRCLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Invoice UBL Mass Download",
```

Comment:
No description field here—should there be?

Reply:
As there is a README file, description is not needed. It will be filled by the README automatically. :)